mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -179,8 +183,13 @@
         Set to <Null />
       </ButtonMenuItem>
       {#if linkedRecordHref}
-        <LinkMenuItem icon={iconLinkToRecordPage} href={linkedRecordHref}>
-          Go To Linked Record
+        <LinkMenuItem icon={iconRecord} href={linkedRecordHref}>
+          Open {$recordSummaries.get(String(column.id))?.get(String(value))}
```

Comment:
Here we need to make the following changes:

- Render the record summary within a `LinkedRecord` component (passing the string into the `recordSummary` prop). That will make it display with the roper background, and also handle tricky cases like this where the record summary contains a `NULL` value that needs to be stylized.

    ![image](https://user-images.githubusercontent.com/42411/215161473-c01be439-8ded-4bb1-8ef3-13c67d9c373d.png)

Reply:
@seancolsen Thank you for the review! I can't believe I didn't think about replacing the `<ButtonMenuItem>` with a `<LinkMenuItem>`. I implemented all of the changes but was having a dilemma with the last one. I was able to get the behavior to match what I believe you are looking for, but this required some changes in the `<LinkedRecord>` component. I am including pictures of the two behaviors and their corresponding style changes. I think having the "Open" and the name of the record on the same line looks better, but I feel as if a unilateral change in the styling may have some contagion to usages of this component that I am unaware of. I would love some guidance on this matter. Thanks again.


Behavior 1:
 -This is the behavior that corresponds to keeping the styles as they were before my changes.

<img width="583" alt="Screenshot 2023-01-27 at 9 46 10 PM" src="https://user-images.githubusercontent.com/63031670/215238439-f9f0aa4c-6147-407e-98f9-e2c5452bbf9b.png">

Behavior 2:
 -This is the behavior that corresponds to my style changes.

<img width="572" alt="Screenshot 2023-01-27 at 9 45 54 PM" src="https://user-images.githubusercontent.com/63031670/215238465-3c4319d6-bbb6-4afb-b28d-b466dbf5d047.png">

The specific line I changed to get this behavior is:

<img width="494" alt="Screenshot 2023-01-27 at 9 55 54 PM" src="https://user-images.githubusercontent.com/63031670/215238562-6750298d-b536-4ead-8c4e-0e7086294dff.png">

This is found in the file, 
/mathesar_ui/src/components/LinkedRecord.svelte

I changed the `display: block;` to `display: inline;`